components/edit-collective/EditPayPalAccount.js

```diff
@@ -139,7 +139,7 @@ const EditPayPalAccount = props => {
         </StyledInputField>
 
         <StyledButton type="submit" mt={2} minWidth={150} loading={isCreating}>
-          <FormattedMessage id="collective.connectedAccounts.paypal.button" defaultMessage="Connect PayPal" />
+          <FormattedMessage defaultMessage="Connect {service}" values={{ service: 'PayPal' }} />
```

Comment:
We don't want people to translate services names, so let's use a common string for all these cases